sched/worker/workers/makers.go

```diff
@@ -8,21 +8,22 @@ import (
 	"github.com/scootdev/scoot/runner/execer/execers"
 	"github.com/scootdev/scoot/runner/runners"
 	"github.com/scootdev/scoot/sched/worker"
-	"github.com/scootdev/scoot/snapshot"
+	"github.com/scootdev/scoot/snapshot/snapshots"
 )
 
 // Makes a worker suitable for using as an in-memory worker.
-func MakeInmemoryWorker(node cluster.Node, tmp *temp.TempDir, filer snapshot.Filer) worker.Worker {
+// TODO: these are useful for unit tests but really shoudldn't be used for integration tests...
```

Comment:
Errr, I disagree with this. I think they can be very useful for integration tests, because they limit the scope of what we're using. Can I fix these in another PR?